entity-framework/core/modeling/owned-entities.md

```diff
@@ -0,0 +1,191 @@
+
+``` csharp
+public class Order
+{
+    public int Id { get; set; }
+    public StreetAddress ShippingAddress { get; set; }
+    public StreetAddress BillingAddress { get; set; }
+}
+```
+
+In order to understand how EF Core will distinguish tracked instances of these objects, it may be useful to think that the defining navigation has become part of the key of the instance alongside the value of the key of the owner and the .NET type of the owned type.
+
+## Nested owned types
+
+In this example OrderDetails owns BillingAddress and ShippingAddress, which are both StreetAddress types. Then OrderDetails is owned by the Order type.
+
+``` csharp
+public class Order
+{
+    public int Id { get; set; }
+    public OrderDetails OrderDetails { get; set; }
+    public OrderStatus Status { get; set; }
+}
+
+public class OrderDetails
+{
+    public StreetAddress BillingAddress { get; set; }
+    public StreetAddress ShippingAddress { get; set; }
+}
+
+public class StreetAddress
+{
+    public string Street { get; set; }
+    public string City { get; set; }
+}
+```
+
+It is possible to chain the `OwnsOne` method in a fluent mapping to configure this model:
+
+``` csharp
+modelBuilder.Entity<Order>().OwnsOne(p => p.OrderDetails, od =>
+    {
+        od.OwnsOne(c => c.BillingAddress);
+        od.OwnsOne(c => c.ShippingAddress);
+    });
+```
+
+Of course, it would be possible to achieve the same thing using `OwnedAttribute`.
+
+In addition to nested owned types, an owned type can reference a regular entity. In the following example, Country is a regular (i.e. non-owned) entity:
+
+``` csharp
+public class StreetAddress
+{
+    public string Street { get; set; }
+    public string City { get; set; }
+    public Country Country { get; set; }
+}
+```
+
+This capability sets owned entity types apart from complex types in EF6.
+
+## Storing owned types in separate tables
+
+Also unlike EF6 complex types, owned types can be stored in a separate table from the owner. In order to override the convention that maps an owned type to the same table as the owner, you can simply call `ToTable` and provide a different table name. The following example will map OrderDetails and its two addresses to a separate table from Order:
+
+``` csharp
+modelBuilder.Entity<Order>().OwnsOne(p => p.OrderDetails, od =>
+    {
+        od.OwnsOne(c => c.BillingAddress);
+        od.OwnsOne(c => c.ShippingAddress);
+    }).ToTable("OrderDetails");
+```
+
+## Querying owned types
+
+When querying the owner the owned types will be included by default. It is not necessary to use the `Include` method, even if the owned types are stored in a separate table. Based on the model described before, the following query will pull Order, OrderDetails and the two owned StreeAddresses for all pending orders:
+
+``` csharp
+var orders = context.Orders.Where(o => o.Status == OrderStatus.Pending);
+```  
+
```

Comment:
It might be worth also showing how to handle the owned entity state:
```C#
var ownedEntry = context.Entry(order).Reference(e => e.BillingAddress).TargetEntry;
ownedEntry.State = EntityState.Unchanged;
ownedEntry.Property(e => e.Street).IsModified = true;
```